src/Softeq.NetKit.Notifications.Web/appsettings.Development.json

```diff
@@ -26,6 +26,13 @@
         "FromEmail": "notifications@softeq.com",
```

Comment:
It's better to remove this email from config.

Reply:
removed